diff --git a/js/public/Debug.h b/js/public/Debug.h
--- a/js/public/Debug.h
+++ b/js/public/Debug.h
@@ -137,11 +137,11 @@
   js::Debugger* debugger;
 
   // Check that |thing| is in the same compartment as our debuggerObject. Used
   // for assertions when constructing BuiltThings. We can overload this as we
   // add more instantiations of BuiltThing.
-#if DEBUG
+#ifdef DEBUG
   void assertBuilt(JSObject* obj);
 #else
   void assertBuilt(JSObject* obj) {}
 #endif
 

